tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```diff
@@ -1134,6 +1134,12 @@ em {
   padding: 0.5em 0.75em;
 }
 
+.subtle {
```

Comment:
This also needs to be added to the generic stylesheet used for the old chat. Search for one of the other spans in this file and you should find it.

Reply:
Done and added the testing evidence to it